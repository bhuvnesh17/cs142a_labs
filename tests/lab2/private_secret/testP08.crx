// test declarations

var id : type;
array a  : void[3];
array a : void[3] [4] [6    ];

func id(id1:void,id2:void,id3:void) : type {}

