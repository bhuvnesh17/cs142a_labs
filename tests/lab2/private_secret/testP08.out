PROGRAM
  DECLARATION_LIST
    DECLARATION
      VARIABLE_DECLARATION
        TYPE
    DECLARATION
      ARRAY_DECLARATION
        TYPE
    DECLARATION
      ARRAY_DECLARATION
        TYPE
    DECLARATION
      FUNCTION_DEFINITION
        PARAMETER_LIST
          PARAMETER
            TYPE
          PARAMETER
            TYPE
          PARAMETER
            TYPE
        TYPE
        STATEMENT_BLOCK
          STATEMENT_LIST

